docker-process.el

```diff
@@ -36,6 +36,11 @@
   :group 'docker
   :type 'boolean)
 
+(defcustom docker-message-commands t
```

Comment:
Naming vetos welcome

Reply:
Please rename to `docker-show-messages`.